pkg/game/elements/placed_tile.go

```diff
@@ -26,6 +26,44 @@ func (pos Position) Y() int16 {
 	return pos.y
 }
 
+func (pos Position) Add(other Position) Position {
+	return NewPosition(pos.x+other.x, pos.y+other.y)
+}
+
+func PositionFromSide(side sideMod.Side) Position { //nolint:gocyclo // similar problem to sides
```

Comment:
This function should use bitmask operations, as it will also make nolint directive redundant

Reply:
is it ok now?